src/main/java/au/com/rayh/XCodeBuilder.java

```diff
@@ -188,23 +199,15 @@
      */
     public final String ipaManifestPlistUrl;
 
-    /**
-     * This is a work around for the broken Apple script as documented here:
-     http://stackoverflow.com/questions/32504355/error-itms-90339-this-bundle-is-invalid-the-info-plist-contains-an-invalid-ke
-     http://stackoverflow.com/questions/32763288/ios-builds-ipa-creation-no-longer-works-from-the-command-line/32845990#32845990
-     http://cutting.io/posts/packaging-ios-apps-from-the-command-line/
-     */
-    public final Boolean signIpaOnXcrun;
-
     // Fields in config.jelly must match the parameter names in the "DataBoundConstructor"
     @DataBoundConstructor
     public XCodeBuilder(Boolean buildIpa, Boolean generateArchive, Boolean cleanBeforeBuild, Boolean cleanTestReports, String configuration,
```

Comment:
This change seems dangerous and will break the compatibility AFAIR (cc @jenkinsci/code-reviewers ). A new Constructor must be added and the old one must be deprecated to allow a smooth upgrade.